### PR TITLE
Add Base58 Flickr & Z85

### PR DESCRIPTION
**Pull request by SkeletalDemise**
Add ability to decode Base58 Flickr and ZeroMQ's Z85 into the files.

### CODE REVIEW

File: ciphey/basemods/Decoders/base58_flicker.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
+import base58
+
+
+@registry.register
+class Base58_flickr(Decoder[str, str]):
```
Comment (lukasgabriel) [inline ciphey/basemods/Decoders/base58_flicker.py:9]: Change the filename so it's the same as the class name to avoid confusion

**Comment by lukasgabriel** [comment]
Don't forget to add it to the init.py once everything is done :p

File: ciphey/basemods/Decoders/base58_flicker.py (hunk 1 above)
Comment (SkeletalDemise) [reply ciphey/basemods/Decoders/base58_flicker.py:9]: The reason I did that was I thought that all the filenames are supposed to be in lowercase, no?

Comment (SkeletalDemise) [reply ciphey/basemods/Decoders/base58_flicker.py]: Never mind, I see I made a typo in the filename. I fixed it.

**Review by lukasgabriel — APPROVED** [review]

**Pull request merged**